addons/website/tests/test_widgets.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import HttpCase, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestWidgets(HttpCase):
```

Comment:
Add it with the other tests, no point in creating a new file for this new test only (either UI or snippets). We'll probably refactor that at some point to make it more consistent.